server/src/main/java/org/opensearch/rest/action/admin/indices/RestIndexPutAliasAction.java

```diff
@@ -120,10 +121,16 @@ public RestChannelConsumer prepareRequest(final RestRequest request, final NodeC
                                     searchRouting = parser.textOrNull();
                                 } else if ("is_write_index".equals(currentFieldName)) {
                                     writeIndex = parser.booleanValue();
+                                } else if ("is_hidden".equals(currentFieldName)) {
```

Comment:
Thanks a lot @gaobinlong for catching it, I think we also miss `must_exist`, right?

Reply:
Thanks @reta, but only `remove` alias supports `must_exist`, this API only supports adding or updating alias, so `must_exist` is not needed. See https://github.com/opensearch-project/OpenSearch/blob/6b8b3efe01a62c221f308a2e3b019d75a7f5ad8a/server/src/main/java/org/opensearch/action/admin/indices/alias/IndicesAliasesRequest.java#L222.

Reply:
Ah got it, thanks!